packages/app-store/office365video/api/callback.ts

```diff
@@ -101,5 +102,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   });
 
   const state = decodeOAuthState(req);
-  return res.redirect(getSafeRedirectUrl(state?.returnTo) ?? "/apps/installed");
+  return res.redirect(
+    getSafeRedirectUrl(state?.returnTo) ?? getInstalledAppPath({ variant: "conferencing", slug: "msteams" })
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.